src/Workspaces/Core/Portable/Storage/AbstractPersistentStorageService.cs

```diff
@@ -249,6 +254,15 @@ public void Dispose()
                 _storage.Dispose();
             }
 
+            public Task<Checksum> ReadChecksumAsync(string name, CancellationToken cancellationToken = default)
```

Comment:
❗️ Do not use optional cancellation tokens for internal APIs

Reply:
Done